robottelo/constants.py

```diff
@@ -728,3 +728,12 @@
               'Certified Cloud Providers (RH CCP)'),
     'usgcb': 'United States Government Configuration Baseline (USGCB)',
 }
+
+SEARCH_EXCEPTIONS_LIST = [
+    'Location',
+    'Organization',
```

Comment:
Maybe yes, but `is_katello = False` for it as it has old style for search bar and search button